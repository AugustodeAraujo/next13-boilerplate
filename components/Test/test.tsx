import { render, screen } from '@testing-library/react'

import Test from '.'

describe('<Test/>', () => {
  it('should render the headi ng', () => {
    const { container } = render(<Test />)

    expect(
      screen.getByRole('heading', { name: /react avan√ßado/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
