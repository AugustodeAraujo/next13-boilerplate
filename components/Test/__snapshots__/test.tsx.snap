// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Test/> should render the headi ng 1`] = `
<div>
  <h1>
    react avan√ßado teste
  </h1>
</div>
`;
